numba/cuda/tests/cudapy/test_gufunc.py

```diff
@@ -121,19 +88,7 @@ def matmulcore(A, B, C):
 
     def test_gufunc_new_axis(self):
 
-        @guvectorize([void(float64[:, :], float64[:, :], float64[:, :])],
```

Comment:
This variant has a different signature to the others, using `float64` instead of `float32`.